From 2ddc5662ea88d2c5ee32b04b4b5b4969572f86d0 Mon Sep 17 00:00:00 2001
From: sadatdaniel <sadatdaniel@yahoo.com>
Date: Mon, 31 Aug 2026 13:23:59 +0000
Subject: [PATCH 1/2] fstab: drop the /product first_stage_mount entries

This device has no /product partition. Android init treats a missing
first_stage_mount entry as fatal, so both the ext4 and erofs /product lines
have to go for the boot to proceed.
---
 tools/make/ramdisk/fstab.exynos9610 | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/make/ramdisk/fstab.exynos9610 b/tools/make/ramdisk/fstab.exynos9610
index 036a007bb..9a0834d84 100644
--- a/tools/make/ramdisk/fstab.exynos9610
+++ b/tools/make/ramdisk/fstab.exynos9610
@@ -7,8 +7,6 @@
 /dev/block/platform/13520000.ufs/by-name/system     /system     erofs   ro  wait,first_stage_mount
 /dev/block/platform/13520000.ufs/by-name/vendor     /vendor     ext4    ro  wait,first_stage_mount
 /dev/block/platform/13520000.ufs/by-name/vendor     /vendor     erofs   ro  wait,first_stage_mount
-/dev/block/platform/13520000.ufs/by-name/product    /product    ext4    ro  wait,first_stage_mount
-/dev/block/platform/13520000.ufs/by-name/product    /product    erofs   ro  wait,first_stage_mount
 
 /dev/block/platform/13520000.ufs/by-name/cache      /cache              ext4	noatime,nosuid,nodev,noauto_da_alloc,discard,journal_checksum,data=ordered,errors=panic    wait,check
 /dev/block/platform/13520000.ufs/by-name/userdata   /data               ext4	noatime,nosuid,nodev,noauto_da_alloc,discard,journal_checksum,data=ordered,errors=panic    latemount,wait,check,reservedsize=128M
-- 
2.34.1

